frontend/package.json

```diff
@@ -40,7 +40,7 @@
 		"eslint-plugin-svelte": "^2.33.0",
 		"postcss": "^8.4.29",
 		"prettier": "^3.0.3",
-		"prettier-plugin-svelte": "^3.0.3",
+		"prettier-plugin-tailwindcss": "^0.5.6",
```

Comment:
Don't we need both plugins?

Reply:
the svelte one comes bundled in this one